### PR TITLE
fixed to crate release notes for metadata files

### PR DESCRIPTION
**Pull request by moishce**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/30009

## Description
fixed the update-release-notes to create release notes Even if the change is only in the metadata files

## Screenshots
Paste here any images that will help the reviewer

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: demisto_sdk/commands/validate/validate_manager.py [hunk 1]
```diff
@@ -715,21 +715,22 @@ def validate_no_duplicated_release_notes(self, added_files):
...
+            changed_meta_files (set): a set of modified metadata files.
 
         Returns:
             bool. True if no missing RN found, False otherwise
         """
         click.secho("\n================= Checking for missing release notes =================\n", fg="bright_cyan")
 
         # existing packs that have files changed (which are not RN, README nor test files) - should have new RN
-        changed_files = modified_files.union(old_format_files).union(added_files)
+        changed_files = modified_files.union(old_format_files).union(added_files).union(changed_meta_files)
```
Comment (ShahafBenYakir) [inline demisto_sdk/commands/validate/validate_manager.py:733]: From this, it looks like we weren't considering changes for metadata files when checking for RN. Isn't this contradict the issue?

File: demisto_sdk/tests/test_files/content_repo_example/Tools/test/assign.ps1 [hunk 2]
```diff
@@ -1,27 +1,27 @@
...
+}
+
+$assignment = New-ManagementRoleAssignment -Role $role -User:$username
+
+if (!$assignment){
+    $msg ="Failed to assign '$username' to role: '$role'"
+    throw $msg
+}
+
+Remove-PSSession $remoteSession
```
Comment (ShahafBenYakir) [inline demisto_sdk/tests/test_files/content_repo_example/Tools/test/assign.ps1:27]: What are these changes?

File: demisto_sdk/tests/integration_tests/update_release_notes_integration_test.py [hunk 3]
```diff
@@ -330,16 +330,44 @@ def test_update_release_notes_modified_apimodule(demisto_client, repo, mocker):
...
     Given
-    - --all flag
+    - --all flag But there are no changes
 
     When
     - Running demisto-sdk update-release-notes command.
 
     Then
-    - Ensure release notes are detected.
+    - Ensure not find changes which would belong in release notes .
```
Comment (ShahafBenYakir) [inline demisto_sdk/tests/integration_tests/update_release_notes_integration_test.py:370]: What does this test actually test?

File: demisto_sdk/tests/test_files/content_repo_example/Tools/test/assign.ps1 (hunk 2 above)
Comment (moishce) [reply demisto_sdk/tests/test_files/content_repo_example/Tools/test/assign.ps1:27]: There is no change here

File: demisto_sdk/tests/integration_tests/update_release_notes_integration_test.py (hunk 3 above)
Comment (moishce) [reply demisto_sdk/tests/integration_tests/update_release_notes_integration_test.py:370]: What happens if there is a name of the pack (which should have changes) and there is no change in the files

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,6 +7,8 @@
 * Fixed an issue in the **lint** command where *finally* clause was not supported in main function.
 * Fixed an issue in the **validate** command where changing any entity ID was not validated.
 * Fixed an issue in the **validate** command where *--staged* flag did not bring only changed files.
+* Fixed the **update-release-notes** to not bump version for changes in the metadata file.
```
Comment (ShahafBenYakir) [inline CHANGELOG.md:10]:
```suggestion
* Fixed the **update-release-notes** to ignore changes in the metadata file.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -7,6 +7,8 @@
...
+* Fixed the **validate** command to not ask for bump version if the change is only in the metadata.
```
Comment (ShahafBenYakir) [inline CHANGELOG.md:11]:
```suggestion
* Fixed the **validate** command to ignore metadata changes when checking if a version bump is needed.
```

**Comment by ShahafBenYakir** [comment]
@moishce Let's make the checks pass.

File: demisto_sdk/commands/common/tools.py [hunk 6]
```diff
@@ -654,6 +671,29 @@ def get_pack_names_from_files(file_paths, skip_file_types=None):
...
+    Returns:
+    files (set): list of files whiteout the types to skip
+    """
+    if file_paths is None:
+        file_paths = set()
+    files = set()
+    for path in file_paths:
+        # renamed files are in a tuples - the second element is the new file name
+        if isinstance(path, tuple):
+            path = path[1]
```
Comment (ShahafBenYakir) [inline demisto_sdk/commands/common/tools.py:690]: Why did we add this again?

Comment (moishce) [reply demisto_sdk/commands/common/tools.py:690]: Because when you rename a file, the "get_modified_and_added_files" function returns it as a tuple

Comment (ShahafBenYakir) [reply demisto_sdk/commands/common/tools.py:690]: 👍

**Review by ShahafBenYakir — APPROVED** [review]

**Pull request merged**